README.md

```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Kanban board: Car collector
 
+> Customers who wish to rent a car can use CarCollectors' website. Using the Vehicles API, the visitor may explore a vast selection of stunning cars. It also allows users to like, comment on, and reserve a certain automobile. The Involvement API is used to save all data.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
+- javascript
+- jest
+- webpack
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://rizwan-learn.github.io/mod2-Kanban/) <br>
+[Live Demonstration Link](https://www.youtube.com/watch?v=mR17kJi45D0)
```

Comment:
I can see you did the requirement! It's working perfectly. 
[OPTIONAL] I would suggest you create a new section and put the link in there
```suggestion
## Presentation
[Live Demonstration Link](https://www.youtube.com/watch?v=mR17kJi45D0)
```